app/queries/lookup_by_base_paths.rb

```diff
@@ -0,0 +1,57 @@
+module Queries
+  module LookupByBasePaths
+    def self.call(base_paths)
+      base_paths_and_content_ids = Edition.with_document
+        .left_outer_joins(:unpublishing)
+        .left_outer_joins(:access_limit)
+        .where(base_path: base_paths)
+        .where("access_limits.edition_id IS NULL")
+        .where("state != 'superseded'")
```

Comment:
touch more Railsy to be `.where.not(state: "superseded")` you might want that to actually be `.where.not(content_store: nil)` though